.github/workflows/ci.yml

```
@@ -75,16 +75,39 @@ jobs:
             sudo journalctl -l --since today
             KITCHEN_LOCAL_YAML=kitchen.dokken.yml /usr/bin/kitchen exec ${{ matrix.suite }}-${{ matrix.os }} -c "journalctl -l"
 
+  integration-windows:
+    needs: [mdl, yamllint, delivery]
+    runs-on: windows-latest
+    strategy:
+      matrix:
+        os:
+          - windows-latest
+        suite:
+          - resources
+      fail-fast: false
+
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v2
+      - name: Install Chef
+        uses: actionshub/chef-install@main
+      - name: test-kitchen
+        uses: actionshub/test-kitchen@main
+        env:
+          CHEF_LICENSE: accept-no-persist
+          KITCHEN_LOCAL_YAML: kitchen.exec.yml
+          ACTIONS_ALLOW_UNSECURE_COMMANDS: 'true'  # allow path editing
```

```suggestion
          ACTIONS_ALLOW_UNSECURE_COMMANDS: true  # allow path editing
```

this should be a bool rather than string 👍🏼